discover/deployment/articles/01-installing-search-engine/03-installing-solr/01-installing-solr-basic.markdown

```diff
@@ -0,0 +1,192 @@
+# Installing Solr: Basic Installation [](id=installing-solr-basic-installation)
+
+There are two ways to install the Liferay Connector to Solr 7:
+
+1. Navigate to the 
+   [Liferay Marketplace](https://web.liferay.com/marketplace/)
+   website and download the LPKG file for Liferay Connector to Solr 7. Once you
+   do, copy the LPKG to your `Liferay_Home/osgi/marketplace` folder.
+
```

Comment:
For future reference, please provide the link to the software on marketplace that you're documenting instead of a general link to Marketplace. It's much friendlier than making people have to search.